app/controllers/api/service_dialogs_controller.rb

```diff
@@ -18,7 +18,8 @@ def fetch_service_dialogs_content(resource)
     end
 
     def create_resource(_type, _id, data)
-      DialogImportService.new.import(data)
+      dialog = DialogImportService.new.import(data)
+      fetch_service_dialogs_content(dialog)
```

Comment:
don't we want to return both the resource + resource.content (i.e. as if we're specifying additional_attributes of content ?)

Reply:
@abellotti `fetch_` are getters for virtual attributes - and it works as expected on `:index` and `:show`, as specified on line 3. This, however, is for `create` and `update`. I played around with this quite a bit today, and unfortunately applying the logic of line 3 to `create` and `update` with additional attributes will require a bit of refactoring in renderer and virtual attribute selection. i'd be happy to go forward with that work, but want to get your thoughts on whether that would be necessary.